atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -135,10 +134,10 @@ def run(self):
 
         # Always set preferred release label - other will be set if old-style
         # label is present
-        release_labels = get_all_label_keys('release')
-        preferred_release_label = get_preferred_label_key(dockerfile_labels,
-                                                          'release')
-        old_style_label = get_all_label_keys('com.redhat.component')[1]
+        release_labels = LabelKeys.get_all_label_keys('release')
+        preferred_release_label = LabelKeys.get_preferred_label_key(dockerfile_labels,
+                                                                    'release')
+        old_style_label = LabelKeys.get_all_label_keys('com.redhat.component')[1]
```

Comment:
this seems wrong to me

it should read old_style 'release' labels instead

Reply:
That's a straight copy of the code with the addition of the wrapper class for flexmock. There's no semantic change there - if it's wrong, it's wrong in the original code.

Reply:
I'm just repeating the existing code and adding a call to my class wrapper so I can flexmock it later. There's no semantic change there, so if there's an error, it's in the original code.

Reply:
yes, I know, I am just pointing it out, that we might need to change it